routes/dish/router.go

```diff
@@ -0,0 +1,58 @@
+package dishRouter
+
+import (
+	"Bete/pkg/httputils"
+	"Bete/services/Dish"
+	"github.com/go-chi/chi/v5"
+	"go.uber.org/fx"
+	"net/http"
+	"strconv"
+)
+
+type params struct {
+	fx.In
+	DishService dishService.Service
+}
+
+type Router interface {
+	RegisterRoutes() chi.Router
+}
+
+type dishRouter struct {
+	dishService dishService.Service
+}
+
+func New(p params) Router {
+	return &dishRouter{
+		dishService: p.DishService,
+	}
+}
+
+func (d dishRouter) RegisterRoutes() chi.Router {
+	router := chi.NewRouter()
+	router.Get("/{id}", d.getDish)
+	router.Post("/", d.createDish)
+	return router
+}
+
+func (r dishRouter) getDish(w http.ResponseWriter, req *http.Request) {
```

Comment:
I don't see the client will need to fetch a dish by id anywhere.

Reply:
Not needed for now , but just in case will remove it later if not needed